backend/uptobox/uptobox.go

```diff
@@ -214,7 +214,7 @@ func NewFs(ctx context.Context, name string, root string, config configmap.Mappe
 
 	client := fshttp.NewClient(ctx)
 	f.srv = rest.NewClient(client).SetRoot(apiBaseURL)
-	f.IDRegexp = regexp.MustCompile("https://uptobox.com/([a-zA-Z0-9]+)")
+	f.IDRegexp = regexp.MustCompile("^https://uptobox\\.com/([a-zA-Z0-9]+)")
```

Comment:
This isn't quite the same regexp as it was - the original didn't have a `^` at the start?

This would look much nicer in backquotes now it has backslashes in!

Reply:
@ncw Done